plugins/org.jkiss.dbeaver.launcher/src/org/eclipse/equinox/launcher/Constants.java

```diff
@@ -0,0 +1,149 @@
+/*
```

Comment:
Can we just import all this stuff from original launcher and not copy-paste it?
We have it in plugin list anyway.

Reply:
I don't think so we targeted to reimplement private method 
` private URL buildLocation(String property, URL defaultLocation, String userDefaultAppendage) `

Reply:
Yes. But why can't we just import constants, JNI bridge and other stuff which is simply copy-pasted?

Reply:
Done, but one thing the equinox classes is in internal package.